### ISSUE TITLE
  could not find function "update_data"

### ISSUE THREAD

**Issue by FrancyJGLisboa** [ModelOriented/forester#9]
The following code snippet in a fresh R and RStudio installation with all forester dependencies in.
library(forester)
data("apartments", package = 'DALEX') 
best_model <- forester(data = apartments,
                       target = "m2.price", 
                       type = "regression",
                       metric = "rmse",
                       tune = FALSE)

Returns:
CREATING MODELS
--- Ranger model has been created ---
Parameter 'cat_features' is meaningless because column types are taken from data.frame.
Please, convert categorical columns to factors manually.
--- Catboost model has been created ---
--- Xgboost model has been created ---
--- LightGBM model has been created ---
__________________________
COMPARISON
Error in update_data(m, data_test[, -which(names(data_test) == target)],  : 
  could not find function "update_data"

**Comment by FrancyJGLisboa** [comment]
Nevermind!
I notice that the examples are incomplete as library(DALEX) is necessary to prevent the mentioned error.
